e2e/solid-router/basic-file-based/src/routes/params-ps/named/{$foo}suffix.tsx

```
@@ -0,0 +1,14 @@
+import { createFileRoute } from '@tanstack/solid-router'
```

ESLint

[error] Expected 1 empty line after import statement not followed by another import.

(import/newline-after-import)